Menus/BuyMenuGUI.cpp

```diff
@@ -1208,20 +1208,45 @@ void BuyMenuGUI::Update()
 
         // Get handle to the currently selected item, if any
         GUIListPanel::Item *pItem = m_pShopList->GetItem(m_ListItemIndex);
+        std::string description = "";
 
-        // Show popup info box next to selected item, but only if it has a description
-        string description = "";
-        // Get it from the regular Entitiy preset
-        if (pItem && pItem->m_pEntity && !pItem->m_pEntity->GetDescription().empty())
+        if (pItem && pItem->m_pEntity) {
             description = pItem->m_pEntity->GetDescription();
-        // Show popup info box next to selected module item, but only if it has a description
-        else if (pItem && pItem->m_ExtraIndex >= 0)
-        {
-            const DataModule *pModule = g_PresetMan.GetDataModule(pItem->m_ExtraIndex);
-            if (pModule && !pModule->GetDescription().empty())
+            const Entity* currentItem = pItem->m_pEntity;
+            const ACraft* pItemAsCraft = dynamic_cast<const ACraft*>(currentItem);
+            if (pItemAsCraft) {
+                int craftMaxPassengers = pItemAsCraft->GetMaxPassengers();
+                int craftMaxMass = static_cast<int>(pItemAsCraft->GetMaxMass()) - static_cast<int>(pItemAsCraft->GetMass());
+                description += "\nMax Mass: " + std::to_string(craftMaxMass) + "\nMax Passengers: " + std::to_string(craftMaxPassengers);
```

Comment:
I wonder if it'd be better to have an extra newline before MaxMass. I guess I should actually look at it before I say anything haha.

Reply:
I've uploaded screenshots below. Let me know what you think.